README.md

```diff
@@ -57,6 +57,10 @@ Here is an example how configuration might look like:
 ```toml
 [server]
 port = 8080
+# Bind a specific IP addresses for server ,"*": bind all IP addresses which is default option, localhost or 127.0.0.1  bind a single IPv4 address
+bind_address = "172.20.10.2" 
+# Specify path for reverse proxy and only [A-Za-z0-9]
+path = "test"
```

Comment:
Could you please update the comment that this is related to URL path specifically? Possibly with en example.
(I was confused for a moment why we need both `path` and `data_dir` until I referred to Go's documentation).

Reply:
got it